src/components/FeatureViews/Onboarding/ImportFlow.js

```diff
@@ -43,9 +43,9 @@ const ImportFlow = () => {
 
     const [state, dispatch] = useReducer(reducer, initalState);
 
-
     useEffect(() => {
-        const currentStep = Number(urlQuery.get('step')) - 1;
+        // const currentStep = Number(urlQuery.get('step')) - 1;
+        const currentStep = Number(urlQuery.get('step'));
```

Comment:
we need to do step-1 to convert it to the step indexing so change this back to line 47